### PR TITLE
Add UI for setting HTTP API tokens

### PR DESCRIPTION
**Pull request by smn**
Currently the UI is missing API token support.

### CODE REVIEW

**Comment by smn** [comment]
@hodgestar @jerith @justinvdm ready for review

File: go/apps/http_api/view_definition.py [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+from django import forms
 
+from go.conversation.view_definition import (
+    ConversationViewDefinitionBase, EditConversationView)
+
+
+class TokenForm(forms.Form):
+    api_tokens = forms.CharField(
+            help_text='The access token for this HTTP Conversation.',
+            required=True)
```
Comment (justinvdm) [inline go/apps/http_api/view_definition.py:10]: I think we should either do validation or cleaning up of this field so we can ensure url-safe token values. Not sure if it needs to be done in this PR though.

File: go/apps/http_api/view_definition.py [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+    push_message_url = forms.CharField(
+        help_text='The URL to forward messages to via HTTP POST.',
+        required=False)
+    push_event_url = forms.CharField(
+        help_text='The URL to forward events to via HTPT POST.',
```
Comment (justinvdm) [inline go/apps/http_api/view_definition.py]: Typo :)

File: go/apps/http_api/view_definition.py (hunk 1 above)
Comment (justinvdm) [inline go/apps/http_api/view_definition.py:10]: On second thought, maybe cleaing up or validating this field isn't too important.

**Comment by justinvdm** [comment]
Other than the typo, :+1:. We can always consider generating the tokens ourselves instead of requiring the user to do that in a later PR.

**Comment by hodgestar** [comment]
:+1:

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**